src/main/java/io/github/thebusybiscuit/slimefun4/core/guide/options/LearningAnimationOption.java

```diff
@@ -0,0 +1,67 @@
+package io.github.thebusybiscuit.slimefun4.core.guide.options;
+
+import java.util.List;
+import java.util.Optional;
+
+import javax.annotation.Nonnull;
+
+import org.bukkit.Material;
+import org.bukkit.NamespacedKey;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+
+import io.github.thebusybiscuit.cscorelib2.data.PersistentDataAPI;
+import io.github.thebusybiscuit.cscorelib2.item.CustomItem;
+import io.github.thebusybiscuit.slimefun4.api.SlimefunAddon;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+
+public class LearningAnimationOption implements SlimefunGuideOption<Boolean> {
+
+    @Nonnull
+    @Override
+    public SlimefunAddon getAddon() {
+        return SlimefunPlugin.instance();
+    }
+
+    @Nonnull
+    @Override
+    public NamespacedKey getKey() {
+        return new NamespacedKey(SlimefunPlugin.instance(), "research_learning_animation");
+    }
+
+    @Nonnull
+    @Override
+    public Optional<ItemStack> getDisplayItem(Player p, ItemStack guide) {
+        if (SlimefunPlugin.getRegistry().isLearningAnimationDisabled()) {
+            return Optional.empty();
+        } else {
+            String enabled = getSelectedOption(p, guide).orElse(true) ? "enabled" : "disabled";
+            List<String> lore = SlimefunPlugin.getLocalization().getMessages(
+                    p, "guide.options.learning-animation." + enabled + ".text");
+            lore.add("");
+            lore.add("&7\u21E8 " + SlimefunPlugin.getLocalization().getMessage(
+                    p, "guide.options.learning-animation." + enabled + ".click"));
+            ItemStack item = new CustomItem(Material.PAPER, lore);
```

Comment:
If we wanna be fancy, we could switch the item here if it is enabled or disabled :o